### ISSUE TITLE
Quake 3 not opening

### ISSUE THREAD

**Issue by corvus2219** [krishenriksen/AnberPorts#70]
I've put the .pk3 files in the right directory like it said but when I click quake 3 in ports it just takes me to Anberports I'm so so confused

**Comment by szalik-rg351** [comment]
arkos or 351elec

**Comment by corvus2219** [comment]
Sorry, ArkOS latest version

**Comment by krishenriksen** [comment]
What do you mean, it takes you to AnberPorts ?

**Comment by corvus2219** [comment]
ArkOS 1.6

Anberports 1.2.2

It just goes to a black screen with a white line flashing and just never loads, I've done everything correctly I'm sure, I dunno

**Comment by corvus2219** [comment]
Yeah I'm very lost, I put all the pk3 files into the folder created by Anberports but as I said just goes to a black screen with a blinking white line in top left corner

**Comment by matrub81** [comment]
Hello.  I have the same problem too .  black or blue screen with blinking dash in the lower left corner I copied the original files in the baseq3 folder but it doesn't work  in the log file it says it can't find the file or folder in line 15
